torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -24,6 +24,8 @@ struct SmallSizeTPairHash {
 
 // Returns true if the TE fuser supports this conv2d.
 bool conv2dIsSupportedJit(const Node* node);
+// Returns true if the TE fuser supports this conv2d with mkldnn prepacked conv.
+bool mkldnnPrepackedConvIsSupportedJit(const torch::jit::Node* node);
```

Comment:
Nit: let's drop `torch::jit::` for consistency with the code around.

Reply:
Removed `torch::jit::` to align with the code around.